src/app/components/forms/onboarding/follows/follows.vue

```diff
@@ -4,40 +4,20 @@
 			<section class="-message">
 				<h3 class="section-header">
 					<translate>
-						Follow Awesome Stuff
+						Join Interesting Communities
 					</translate>
 				</h3>
 
 				<p class="text-muted">
-					<translate>
-						Get connected right away with some of the best games on Game Jolt.
-					</translate>
+					<span v-translate>
+						We have fanart, lets plays, guides,
+						<i>DANK</i>
```

Comment:
Need corporate approval here.

Reply:
"Explore fan-created artwork, videos, game guides and more."

Reply:
aww